frontend/src/components/requisitionDetail/RequisitionDetail.tsx

```diff
@@ -70,10 +66,14 @@ const RequisitionDetail: React.FC<{}> = (props) => {
         <Col xs={24} sm={24} md={9} lg={9} xl={9}>
           <Title level={3} style={{ fontSize: "20px" }}>Actions</Title>
           <Button type="primary" className="action-button">Approve</Button>
-          <Link to={`${location.pathname.replace(/\/+$/, "")}/edit`}>
-            <Button className="action-button">Edit</Button>
-          </Link>
-          <Button className="action-button" danger>Cancel</Button>
+          {rekData.canEdit
```

Comment:
I'm not saying you should do this here, but one thing to consider going forward is that in certain cases, showing a disabled button with an explanation about why it's disabled ("You need the <permission> permission to do this" / "Only project leads and admins can approve requisitions") can make it more clear to users why they can't see/do a particular action (makes it easier to understand or ask for help).

Reply:
Hmm that's a very good idea, I actually like that a lot. I'll add it in right now

Reply:
Added it in. It looks like this
<img width="721" alt="Screen Shot 2020-07-09 at 11 32 54 PM" src="https://user-images.githubusercontent.com/8273038/87113651-be9ce100-c23c-11ea-8b96-d81ab298c538.png">

Reply:
Oops fixed the typo